.docker/tests/conftest.py

```diff
@@ -42,8 +42,8 @@ def is_container_ready():
         return '✔ broker:' in output and 'Daemon is running' in output
 
     docker_services.wait_until_responsive(
-        timeout=600.0,
```

Comment:
Is this in seconds? If so, are 10 minutes really not enough for the services to start? And if so, what makes you think another 6 is going to cut it?

Reply:
I have no strong clue on why sometime it not finished. I guess the pause is too short that keep on set the commands to the container cause the problem. 
I'll decrease timeout back to 600s and see if the test will pass.

Reply:
Just checked with @mbercx, so probably the problem was he turned VPN on the machine which occasionally bring the issue to the test. As I notice only arm64 test are failed in main after PRs got merged. 
But I think it is still valuable to increase the pause to 2s, it is the pause that send the `verdi status` command to container, 0.1 seems too often.